relayer-cli/tests/integration.rs

```diff
@@ -47,14 +47,18 @@ fn simd_chain() -> CosmosSDKChain {
 #[test]
 #[ignore]
 fn query_connection_id() {
+    /* the current informaldev/simd image has an incompatible (old?) protobuf implementation
```

Comment:
What’s the path forward for re-enabling this test? Do we just need to wait for the image to be updated to the new protobuf implementation? What does that mean actually? New protobuf message descriptors?

Reply:
Yes, we need a new docker image with an updated simd application. The protobuf message was updated in the SDK and an extra field added. We have more recent protobuf structs than the simd. (The simd is from the times of game-of-zones phase 1 when a bunch of the protobuf looked different.)